audformat/core/table.py

```diff
@@ -647,7 +647,17 @@ def load(
             pickled = True
 
         if pickled:
-            self._load_pickled(pkl_file)
+            try:
+                self._load_pickled(pkl_file)
+            except Exception as ex:
```

Comment:
Could you replace `Exception` with the actual exception to not catch keyboard interrupts or similar stuff here.

Reply:
Maybe we could support two different kind of exceptions here. One of the non-matching pickle protocol and the other with a corrupted pickle file as you used in the tests?

Reply:
Ok, replaced it with `ValueError`, which raised if the pickle protocol is not understood, and `EOFError`, which we use in the test.

Reply:
Interesting that they go with `ValueError` for it. But thanks for changing.